packages/async-api-render-engine/README.md

```diff
@@ -18,10 +18,6 @@ The AsyncAPI render engine is based on the [AsyncAPI React](https://github.com/a
   yarn add @kyma-project/dc-async-api-render-engine
   ```
 
-## Prerequisites
```

Comment:
maybe instead of deleting this section add asyncapi-react as prerequisite?

Reply:
but I do not need `asyncapi-react`, like I should not need `odata-react`

Reply:
AsyncAPI is in dependencies array of this render-engine, odata-react in dc-odata doesn't at the moment.